### PR TITLE
Add validation api for content import

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

File: molo/core/content_import/tests/test_content_validation.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        self.create_pages(
+            self.workspace, source=None, count=1, locale='spa_ES',
+            primary_category='an-invalid-primary-category')
+
+        # main language page with invalid primary category
+        self.create_pages(
+            self.workspace, count=1, locale='eng_GB',
+            primary_category='an-invalid-uuid')
+
+        error = ContentImportValidation(self.workspace).is_validate_for([
```
Comment (miltontony) [inline molo/core/content_import/tests/test_content_validation.py]: this should probably be called `errors`

File: molo/core/content_import/validation.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+        if self.validate_wagtail_has_no_language():
+            for l in locales:
+                self.validate_translated_content_has_source(l)
+                self.validate_page_primary_category_exists(l)
+                self.validate_translated_content_source_exists(l)
+
+        return self.errors
+
+    def validate_wagtail_has_no_language(self):
```
Comment (miltontony) [inline molo/core/content_import/validation.py]: What are you trying to test here - you're looping through all the languages and throwing an error

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

**Comment by miltontony** [comment]
:+1: :cake: :star2:

**Pull request merged**